action.yml

```diff
@@ -3,49 +3,58 @@ description: Syncs and merges your current repository with the upstream reposito
 author: Dabreadman
 
 inputs:
-  upstream_repo:
-    description: URL of gitHub public upstream repo
+  upstream_repo_url:
+    description: URL of GitHub public upstream repo
     required: true
   upstream_branch:
     description: Branch to merge from upstream (defaults to downstream branch)
     required: false
     default: "main"
+  downstream_repo_url:
+    description: URL of the GitHub repo from upstream
+    required: true
   downstream_branch:
-    description: Branch to merge into downstream
+    description: Branch to merge into downstream, if not provided it will default to the upstream branch name
```

Comment:
I'd add the following default to the token input, so that it defaults to the pipeline token

```diff
token:
    description: GitHub Bot token
    required: true
+   default: ${{ secrets.GITHUB_TOKEN }} 
```